project/settings.py

```diff
@@ -119,4 +119,7 @@
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 STATIC_URL = '/static/'
-STATIC_ROOT = os.path.join(BASE_DIR, 'static')
\ No newline at end of file
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+MEDIA_ROOT= os.path.join(BASE_DIR, 'media')
+MEDIA_URL= "/media/"
```

Comment:
Remind me what's supposed to be in media?

Reply:
The media root is holding all of the meal photos that I am accessing through the templates.